site/docs/v1/tech/tutorials/integrate-dotnet-api.md

```diff
@@ -0,0 +1,191 @@
+
+The `state` parameter is any information that you would like returned on successful login.
+
+The `scope` is the OAuth scopes you want to request. In this example, the `openid` scope is requested, which is required to get the `app.idt` cookie containing the JWT with the user's claims. You can request other scopes too, such as `offline_access` to get a refresh token, which can be used to get a new access token when the current one expires. The refresh token is stored in a cookie called `app.rt`.
+
+Now add the following to the `profile.html` file:
+
+```html
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-dotnet-api-example/main/LoginPage/profile.html %}
+```
+
+This page is the page that the Hosted Backend API will redirect to after the login flow is complete.
+
+The page first calls the [`/me` route](/docs/v1/tech/apis/hosted-backend#me), which is part of the FusionAuth Hosted Backend API and returns the identity of the currently logged-in user.
+
+The `profile.html` page also has a button that calls the `identity` route of the {{page.technology}} API. Note that the `fetch` call for this button includes the `credentials: 'include'` option. This option tells the browser to include the cookies in the request so that the {{page.technology}} API can read the `app.idt` cookie and extract the JWT.
+
+If the API call is successful, the `identity` route will return the identity and claims of the user. If the API call is not successful, the `identity` route will return a `401 Unauthorized` response. In this case, the code will try refreshing the access token using the refresh token in the `app.rt` cookie. To do this, the code makes a `fetch` request to the [refresh token route](/docs/v1/tech/apis/hosted-backend#refresh) of the Hosted Backend API. If the refresh token is valid, FusionAuth's Hosted Backend API will return a new access token. The code then tries the `identity` route again with the new access token.
+
+
+{% include _callout-note.liquid content="A note on CORS (Cross-Origin Resource Sharing): For this setup to work, all components, including the web page, the .NET Core API, and FusionAuth, must be on the same domain. This is because the `app.idt` JWT cookie and `app.rt` refresh cookie are set to the same domain as the FusionAuth instance. Since everything runs on the same domain, CORS won't usually be an issue. However, you'll be running all of this on `localhost` to test, with each component running on a different port, which will cause CORS issues. For this reason, we need to enable CORS on FusionAuth for the login webpage to access the FusionAuth Hosted Backend API. To do this, navigate to 'Settings', then 'System' on the sidebar, and then the 'CORS' tab. Enable CORS, and check 'GET', 'POST', and 'OPTIONS'. Enable 'Allow Credentials', and  add `http://localhost:3000` to the list of allowed origins." %}
```

Comment:
Please use the `<span>text<span>{:.css-class}` syntax for navigation and UI elements, e.g. `<span>Settings</span>{:.breadcrumb}` and `<span>Allow Credentials</span>{:.uielement}`